### PR TITLE
Added EMCS messages. Also removed some other messages that are not par…

### PR DESCRIPTION
**Pull request by mauro-fanton**
…t of this delivery

### CODE REVIEW

File: source/documentation/partials/_IE906.md
Comment (mpbostock) [inline source/documentation/partials/_IE906.md:1]: Can we remove this file? It's not in the list of IE messages we need to know about

File: source/documentation/partials/_IE917.md
Comment (mpbostock) [inline source/documentation/partials/_IE917.md:1]: Can we remove this file? It's not in the list of IE messages we need to know about

File: source/documentation/partials/_IE928.md
Comment (mpbostock) [inline source/documentation/partials/_IE928.md:1]: Can we remove this file? It's not in the list of IE messages we need to know about

File: python/__pycache__/messages.cpython-38.pyc
Comment (mpbostock) [inline python/__pycache__/messages.cpython-38.pyc:1]: We should probably remove this file and add something to the gitignore to ignore anything in a __pycache__ folder??

File: python/__pycache__/utils.cpython-38.pyc
Comment (mpbostock) [inline python/__pycache__/utils.cpython-38.pyc:1]: We should probably remove this file and add something to the gitignore to ignore anything in a __pycache__ folder??

**Review by mpbostock — CHANGES_REQUESTED** [review]
Can the messages.py be run and the new files added please? IE815 is still missing for example

File: source/documentation/partials/_IE917.md
Comment (mauro-fanton) [reply source/documentation/partials/_IE917.md:1]: cannot see this in the PR. should have been removed.

File: source/documentation/partials/_IE928.md
Comment (mauro-fanton) [reply source/documentation/partials/_IE928.md:1]: cannot see this in the PR. should have been removed.

**Comment by mauro-fanton** [comment]
forgot to add , commit and push he IE815

**Comment by mauro-fanton** [comment]
The me messages.py get run as part of messagetypes.py

File: python/messagetypes.py
Comment (jlid87) [inline python/messagetypes.py:1]: Can we add IE717, IE880?

**Review by mpbostock — DISMISSED** [review]

**Review by mpbostock — APPROVED** [review]

**Pull request merged**